test/integration/src/test/java/com/liferay/faces/portal/test/integration/demo/PrimeFacesUsersPortletTester.java

```diff
@@ -48,506 +52,621 @@
+		// 1. Take note of the screen name of the first user in the list on page 1.
+		String firstScreenNameOnPageOne = browserDriver.findElementByXpath(SCREEN_NAME_CELL_XPATH).getText();
+		logger.debug("firstUserOnPageOne=[{0}]", firstScreenNameOnPageOne);
 
-		// Enter an invalid email address by removing the "@" symbol and click the "Submit" button.
-		browserDriver.clearElement(EMAIL_ADDRESS_FIELD_XPATH);
-		browserDriver.sendKeysToElement(EMAIL_ADDRESS_FIELD_XPATH, "john.adamsliferay.com");
+		// 2. Click the *Next Page* button and verify that the page 2 button is highlighted.
+		browserDriver.clickElement(getNavigationButtonXpath("Next"));
 
-		String submitButtonXpath = "//button[contains(@id, ':pushButtonSubmit') and @type='submit']";
-		browserDriver.clickElement(submitButtonXpath);
+		BrowserStateAsserter browserStateAsserter = getBrowserStateAsserter();
+		browserStateAsserter.assertTrue(pageButtonClassActive(2));
+
+		// 3. Verify that the screen name noted from page 1 is not present on page 2.
+		browserStateAsserter.assertTextNotPresentInElement(firstScreenNameOnPageOne, SCREEN_NAME_CELL_XPATH);
+
+		// 4. Take note of the screen name of the first user in the list on page 2.
+		String firstScreenNameOnPageTwo = browserDriver.findElementByXpath(SCREEN_NAME_CELL_XPATH).getText();
+		logger.debug("firstUserOnPageTwo=[{1}]", firstScreenNameOnPageTwo);
+
+		// 5. Click on *3* button and verify that the page 3 button is highlighted.
+		browserDriver.clickElement(getPageButtonXpath(3));
+		browserStateAsserter.assertTrue(pageButtonClassActive(3));
+
+		// 6. Verify that the screen names noted from page 1 and 2 are not present on page 3.
+		browserStateAsserter.assertTextNotPresentInElement(firstScreenNameOnPageOne, SCREEN_NAME_CELL_XPATH);
+		browserStateAsserter.assertTextNotPresentInElement(firstScreenNameOnPageTwo, SCREEN_NAME_CELL_XPATH);
+
+		// 7. Take note of the screen name of the first user in the list on page 3.
+		String firstScreenNameOnPageThree = browserDriver.findElementByXpath(SCREEN_NAME_CELL_XPATH).getText();
+		logger.debug("firstUserOnPageThree=[{0}]", firstScreenNameOnPageThree);
+
+		// 8. Click on *Previous Page* button and verify that the page 2 button is highlighted.
+		browserDriver.clickElement(getNavigationButtonXpath("Previous"));
+		browserStateAsserter.assertTrue(pageButtonClassActive(2));
+
+		// 9. Verify that the screen names noted from page 1 and 3 are not present on page 2.
+		browserStateAsserter.assertTextNotPresentInElement(firstScreenNameOnPageOne, SCREEN_NAME_CELL_XPATH);
+		browserStateAsserter.assertTextNotPresentInElement(firstScreenNameOnPageThree, SCREEN_NAME_CELL_XPATH);
+
+		// 9. Click on *Last Page* button and verify that the last button is highlighted and that the *Next Page* and
+		// *Last Page* buttons are disabled.
+		browserDriver.clickElement(getNavigationButtonXpath("Last"));
+		browserStateAsserter.assertTrue(textPresentInElementClass("ui-state-active",
```

Comment:
@stiemannkj1 Not sure it would be necessary to append the word "condition" here, since it reads nicely enough as it is, but you could consider it for the sake of consistency.